### PR TITLE
Fixed failing CI tests

### PR DESCRIPTION
**Pull request by omichel**
GitHub Action CI tests are failing due to missing files, bad code format, etc.
This PR attempt to fix them.

### CODE REVIEW

**Comment by omichel** [comment]
(please do not merge this PR unless all the tests are passing)

**Review by medrimonia — APPROVED** [review]

**Comment by medrimonia** [comment]
Thank you very much!

**Comment by omichel** [comment]
All the 44 tests [461/461] are passing.

**Pull request merged**
